src/domain/index.ts

```diff
@@ -28,3 +29,8 @@ export type Header =
       type: "main";
     }
   | PageHeader;
+
+export interface EstimatedWithdrawFee {
```

Comment:
Doesn't this belong to the Hermez domain?

Not very relevant though since now with the library typings we probably will end up merging all domain modules into one.